keanu-project/src/test/java/io/improbable/keanu/vertices/dbl/DoubleVertexTest.java

```diff
@@ -2,17 +2,30 @@
 
 import io.improbable.keanu.tensor.dbl.DoubleTensor;
 import io.improbable.keanu.tensor.dbl.Nd4jDoubleTensor;
+import io.improbable.keanu.vertices.ConstantVertex;
 import io.improbable.keanu.vertices.dbl.nonprobabilistic.ConstantDoubleVertex;
 import io.improbable.keanu.vertices.dbl.probabilistic.GaussianVertex;
 import io.improbable.keanu.vertices.dbl.probabilistic.UniformVertex;
 import io.improbable.keanu.vertices.intgr.IntegerVertex;
+import org.junit.Before;
 import org.junit.Test;
 
+import static io.improbable.keanu.tensor.TensorMatchers.valuesAndShapesMatch;
+import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.Assert.assertArrayEquals;
 import static org.junit.Assert.assertEquals;
 
 public class DoubleVertexTest {
 
+    private DoubleTensor vectorA;
+    private DoubleTensor vectorB;
```

Comment:
Your test code would be more concise if these were vertices, i.e.

    DoubleVertex vectorA = ConstantVertex.of(new double[]{1., 2.}, 2);
    DoubleVertex vectorB = ConstantVertex.of(new double[]{1., 2., 3., 4., 5., 6.}, 6);